morebits.js

```diff
@@ -989,26 +991,32 @@ Morebits.string = {
 		str = str.toString();
 		return str.substr( 0, 1 ).toLowerCase() + str.substr( 1 );
 	},
-	splitWeightedByKeys: function( str, start, end, skip ) {
+
+	/**
+	 * Gives an array of substrings of `str` starting with `start` and 
+	 * ending with `end`, which is not in `skiplist`
+	 * @param {string} str  @param {string} start @param {string} end @param {(array|string)} [skiplist] 
```

Comment:
Since the description makes it clear the functionality of each parameter, I thought I'd rather not waste space putting each on a new line.

Fixed now.